bazel/toolchain/s390x-none-linux-gnu/cc_toolchain_config.bzl

```diff
@@ -0,0 +1,138 @@
+load("@bazel_tools//tools/build_defs/cc:action_names.bzl", "ACTION_NAMES")
+load(
+    "@bazel_tools//tools/cpp:cc_toolchain_config_lib.bzl",
+    "feature",
+    "flag_group",
+    "flag_set",
+    "tool_path",
+)
+
+all_link_actions = [
+    ACTION_NAMES.cpp_link_executable,
+    ACTION_NAMES.cpp_link_dynamic_library,
+    ACTION_NAMES.cpp_link_nodeps_dynamic_library,
+]
+
+all_compile_actions = [
+    ACTION_NAMES.assemble,
+    ACTION_NAMES.c_compile,
+    ACTION_NAMES.clif_match,
+    ACTION_NAMES.cpp_compile,
+    ACTION_NAMES.cpp_header_parsing,
+    ACTION_NAMES.cpp_module_codegen,
+    ACTION_NAMES.cpp_module_compile,
+    ACTION_NAMES.linkstamp_compile,
+    ACTION_NAMES.lto_backend,
+    ACTION_NAMES.preprocess_assemble,
+]
+
+def _impl(ctx):
+    tool_paths = [
+        tool_path(
+            name = "ar",
+            path = "/usr/bin/s390x-linux-gnu-ar",
+        ),
+        tool_path(
+            name = "cpp",
+            path = "/usr/bin/s390x-linux-gnu-cpp",
+        ),
+        tool_path(
+            name = "gcc",
+            path = "/usr/bin/s390x-linux-gnu-gcc",
+        ),
+        tool_path(
+            name = "gcov",
+            path = "/usr/bin/s390x-linux-gnu-gcov",
+        ),
+        tool_path(
+            name = "ld",
+            path = "/usr/bin/s390x-linux-gnu-ld",
+        ),
+        tool_path(
+            name = "nm",
+            path = "/usr/bin/s390x-linux-gnu-nm",
+        ),
+        tool_path(
+            name = "objdump",
+            path = "/usr/bin/s390x-linux-gnu-objdump",
+        ),
+        tool_path(
+            name = "strip",
+            path = "/usr/bin/s390x-linux-gnu-strip",
+        ),
+    ]
+
+    # Update the value of __TOOLCHAIN_SYSROOT__ every time the list
+    # of cxx_builtin_include_directories has been modified.
+    #
+    # This is needed to make bazel realize that the updated toolchain
+    # is different from the previous one and handle caching correctly.
+    #
+    # https://github.com/kubevirt/kubevirt/pull/8404#issuecomment-1275096374
```

Comment:
You haven't defined `__TOOLCHAIN_SYSROOT__`. Not a problem right now, but it would be better to have it right out of the gate so that we don't get tripped up next time the base OS is switched.

Reply:
I can add that, it shouldn't break anything.

Reply:
That's because in that case the toolchain is downloaded separately, and Bazel is able to figure out that changes to the source URL should result in the cache being invalidated. That's not the case for x86_64 and s390x, where the toolchain comes from distro packages.

Reply:
Ah, ok that makes sense. I have added it for now, but i guess that also means i can drop it again when i make the change to having the s390x be in Bazel.

Reply:
I'm not sure if that will be enough to make the define redundant, I would probably still keep it around just to be on the safe side. But we'll cross that bridge when we get to it :)

Reply:
I mean it won't harm anything, but if the new toolchain is defined as a tar, then i would assume that bazel automatically updates the version in cache if some rpm version changes.